sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/FileIndexSuite.scala

```diff
@@ -290,9 +290,9 @@ class FileIndexSuite extends SharedSparkSession {
           }
         }
         HiveCatalogMetrics.reset()
```

Comment:
I think we can remove all changes here. There's no strong preference between `==` and `===` when it compares a scalar value. Sometimes `==` provides a better assert message in this case.

Reply:
Will remove these.